entity-service-impl/src/test/java/org/hypertrace/entity/query/service/EntityQueryConverterTest.java

```diff
@@ -114,4 +115,23 @@ public void test_convertToEdsQuery_functionExpressionOrderByExpression_throwsExc
     assertThrows(UnsupportedOperationException.class,
         () -> EntityQueryConverter.convertToEDSQuery(request, Collections.emptyMap()));
   }
+
```

Comment:
Are there no IntegrationTests for EntityQueryService?
I think we should add one.

Reply:
There weren't any. Let me try to add them

Reply:
added integration tests